### ISSUE TITLE
Switch does not respect clickable/focusable false

### ISSUE THREAD

**Issue by m-radzikowski** [kyleduo/SwitchButton#64]
We have a scenario in which I need to use SwitchButton as indicator, without capability to click on it and change it's state. But click in the area of switch should be handled by component below it, like usual in Android.

I've created two switches, one native and one SwitchButton:
```xml
	<Switch
		android:layout_width="wrap_content"
		android:layout_height="wrap_content"
		android:clickable="false"
		android:focusable="false"/>

	<com.kyleduo.switchbutton.SwitchButton
		android:layout_width="wrap_content"
		android:layout_height="wrap_content"
		android:clickable="false"
		android:focusable="false"/>
```

Click on the native Android switch does nothing to it and is handled by component underneath. But SwitchButton ignores clickable and focusable config and changes state when clicked / moved.

Right now we've to create workaround with listeners to manually prevent state change and pass click event to other view.

Tested on version 1.4.2.

**Comment by kyleduo** [comment]
I'll ckeck it out asap.

**Comment by kyleduo** [comment]
Fixed in 1.4.3. Thanks for pointing out this.
